components/textfield/stories/template.js

```diff
@@ -109,11 +125,11 @@ export const Template = ({
 		}))}
 		${when(multiline,
 			() => html`<textarea
-				placeholder=${ifDefined(placeholder)}
+				placeholder=${ifDefined(placeholder && placeholder.trim() ? placeholder : undefined)}
 				name=${ifDefined(name)}
 				id=${ifDefined(id ? `${id}-input` : undefined)}
 				.value=${ifDefined(value)}
-				autocomplete=${autocomplete ? undefined : "off"}
+				autocomplete=${ifDefined(autocomplete ? undefined : "off")}
 				?required=${isRequired}
 				?disabled=${isDisabled}
```

Comment:
For `required`, I see the attribute getting added, but do we need to represent that somehow in the UI? With the asterisk? I see the * in Figma, but wasn't sure if we needed it here somewhere.

Reply:
I think I would leave that on the implementations to add since there doesn't seem to be any specific CSS needed if the `required` attribute is on. There is [some documentation](https://spectrum.corp.adobe.com/page/text-field/#Required-or-optional) on what styling requirements there are for required text, I think the only thing that might be relevant to text field would be the asterisk in the field label, but that's in a different component.

Reply:
Oh right, in the field label component. Got it.